VultisigApp/VultisigApp/Views/Components/VaultDeletionDetails.swift

```diff
@@ -91,7 +91,7 @@ struct VaultDeletionDetails: View {
     }
     
     private func getVaultType() -> String {
-        return "\(vault.getThreshold() + 1) of \(vault.signers.count) Vaults"
+        return "Part \(vault.getThreshold() + 1) of \(vault.signers.count)"
```

Comment:
This one is wrong , so user want to know which part they are deleting , not the vault type. 
In this case , we should get the index of local party in the vault.signers , instead of get threshold. 

User want to know which part he/she is deleting , like Part1of3 or Part2of3, or Part3of3